client/user.go

```diff
@@ -117,3 +154,103 @@ func (h *V1Client) CreateUser(user *models.V1UserEntity) (string, error) {
 	}
 	return *resp.Payload.UID, nil
 }
+
+// UpdateUser update existing user.
+func (h *V1Client) UpdateUser(uid string, user *models.V1UserUpdateEntity) error {
+	param := clientv1.NewV1UsersUIDUpdateParams().WithUID(uid).WithBody(user)
+	_, err := h.Client.V1UsersUIDUpdate(param)
+	if err != nil {
+		return err
+	}
+	return nil
+}
+
+// AssociateUserProjectRole associate project role for the user.
+func (h *V1Client) AssociateUserProjectRole(userUID string, body *models.V1ProjectRolesPatch) error {
+	param := clientv1.NewV1UsersProjectRolesPutParams().WithUID(userUID).WithBody(body)
+	_, err := h.Client.V1UsersProjectRolesPut(param)
```

Comment:
return h.Client.V1UsersProjectRolesPut(param)

Reply:
remove err check and return err directly